lib/gtfs/calendar.ex

```diff
@@ -0,0 +1,131 @@
+defmodule Gtfs.Calendar do
+  alias Gtfs.Csv
+  alias Gtfs.Service
+
+  @type t :: %{Date.t() => [Service.id()]}
+
+  @typep gtfs_calendar_row :: %{
+           service_id: Service.id(),
+           monday: boolean(),
+           tuesday: boolean(),
+           wednesday: boolean(),
+           thursday: boolean(),
+           friday: boolean(),
+           saturday: boolean(),
+           sunday: boolean(),
+           start_date: Date.t(),
+           end_date: Date.t()
+         }
+
+  @typep gtfs_calendar_date_row :: %{
+           service_id: Service.id(),
+           date: Date.t(),
+           exception_type: :added | :removed
+         }
+
+  @spec from_files(binary() | nil, binary() | nil) :: t()
+  def from_files(calendar_file, calendar_dates_file) do
+    gtfs_calendar = Csv.parse(calendar_file, fn _row -> true end, &calendar_from_csv_row/1)
```

Comment:
I think this case is a little different, since there are two different files, and they're isolated from the other data (meaning they don't need the list of bus route ids or anything).

I just tried moving the parsing into `data.ex`, but it meant exporting 3 functions and 2 types instead of just 1 function (`from_files` vs `gtfs_calendar_row`, `gtfs_calendar_date_row`, `calendar_from_csv_row`, `calendar_date_from_csv_row`, and `calendar_from_gtfs`).

`data.ex` is also bigger and more complicated than `calendar.ex`, and I'd rather keep logic out of it if it's reasonable.

And even with the move, `calendar.ex` still has to understand gtfs and its format, since it still needs to define the parsers.

The inconsistency is a little weird, but I think the rest of the situation makes it worth it to be inconsistent.